api/v1/cluster_types.go

```diff
@@ -2804,6 +2811,59 @@ func (roleConfiguration *RoleConfiguration) GetRoleInherit() bool {
 	return true
 }
 
+// DatabaseConfiguration is the representation in Kubernetes of a PostgreSQL Database
```

Comment:
I know this patch is a prototype.
Just mentioning here, perhaps for a separate ticket, that we're missing the locale and collate settings.

Reply:
Agree. I did that to avoid having too many parameters that are usable only when creating a database.
You can't change them after you create your DB.

The same applies to **encoding**.